README.md

```diff
@@ -43,7 +55,19 @@ The plan frameowork and concept were last updated 2023-03-08
   - [ ] Audit the current 2022 membership token for expiration
   - [ ] Grandfather in relevant members per Operating System Document
   
-  ## Contract Risk Checks
+## MUT8 Date Marker
+The plan frameowork and concept were last updated 2023-03-08
+
+## BOT9 - using lexy or kay to parse ruleset and passively trigger audits
+
+We use the legal tests of:
+- more likely than not
+- on the balance of probabiliies
+- beyond reasonable doubt
+
+by running multiple discord bots and weighing their frame assessment to automate the central accounting problem.
```

Comment:
Do me a favor and lets leave the accounting thing to the side.  I have a model to discuss with you.  We need some more foundational things first.  This is on my mental dev map but we're not ready.